lib/components.js

```diff
@@ -2,8 +2,7 @@
  *  Copyright (c) Coherent Labs AD. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-
-const newLinesRegExp = new RegExp('^\s+|\s+$', 'g');
+const newLinesRegExp = new RegExp('^s+|s+$', 'g');

```

Comment:
Have you tested the new Regexes? I saw that you are removing the backslashes (\), but to my knowledge `s` will only match the S character, while `\s` matches whitespaces.